scipy/sparse/linalg/isolve/tfqmr.py

```diff
@@ -0,0 +1,182 @@
+import warnings
+import numpy as np
+from .utils import make_system
+
+
+__all__ = ['tfqmr']
+
+
+def tfqmr(A, b, x0=None, tol=1e-5, maxiter=None, M=None,
+          callback=None, atol=None, view=False):
+    """
+    Use Transpose-Free Quasi-Minimal Residual iteration to solve ``Ax = b``.
+
+    Parameters
+    ----------
+    A : {sparse matrix, dense matrix, LinearOperator}
```

Comment:
`dense matrix` -> `ndarray` (right?)

Reply:
Note that these fields are type descriptors, so it should use the actual type normally.

Reply:
If it was copied from other code in `sparse.linalg`: most of that code is really old, would be nice to avoid propagating its style.